osgar/tools/lidarview.py

```diff
@@ -313,6 +311,16 @@ def _step(self, direction):
                 self.scan2 = deserialize(data)
             elif stream_id == self.camera_id:
                 self.image = get_image(deserialize(data))
+                current = self.current
+                while current + direction >= 0 and current + direction < len(self.log):
+                    current += direction
+                    _, new_stream_id, new_data = self.log[current]
+                    if new_stream_id == self.bbox_id:
+                        self.bbox.append(deserialize(new_data))
+                    if new_stream_id in self.title_id:
```

Comment:
here I suppose that if you do not use bbox in title you will get it twice into the list (??)

Reply:
I am not sure I follow, if you don't add bbox textual representation to the title, what will show twice in what list?